lib/skate/ueberauth/strategy/fake.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Skate.Ueberauth.Strategy.Fake do
+  use Ueberauth.Strategy
+
+  @impl Ueberauth.Strategy
+  def handle_request!(conn) do
+    conn
+    |> redirect!("/auth/cognito/callback")
+    |> halt()
+  end
+
+  @impl Ueberauth.Strategy
+  def handle_callback!(conn) do
+    conn
+  end
+
+  @impl Ueberauth.Strategy
+  def uid(_conn) do
+    "fake_uid"
+  end
+
+  @impl Ueberauth.Strategy
+  def credentials(_conn) do
+    one_hour_in_seconds = 60 * 60
+    expiration_time = System.system_time(:second) + one_hour_in_seconds
```

Comment:
Will we have to refresh every hour in development? That might be disruptive? Could this be extended?

Reply:
This is one hour after the last time the page was loaded (as the token is updated each time). Seems unlikely to be a problem in practice, but it also doesn't matter what this is.

Reply:
I can imagine opening a bunch of routes, going to a meeting, and wanting to come out of the meeting able to resume without reloading. Let's bump it to 9 hours.

Reply:
Are you sure it's one hour since the page was loaded? I believe this is one hour since the cookie is set, which only happens when you go through the /auth/cognito -> /auth/cognito/callback flow, which only happens when you try to load the page while not being signed in. (i.e. if you come back in 30 mins, the cookie will still be a valid authenticated session, so you won't go through the request -> callback flow.)

Reply:
Oh that's interesting, that sounds true. 9 hours it is.